dubbo-rpc/dubbo-rpc-dubbo/src/test/java/org/apache/dubbo/rpc/protocol/dubbo/DubboProtocolTest.java

```diff
@@ -120,7 +120,7 @@ public void testDubboProtocolWithMina() throws Exception {
         System.out.println(service.stringLength(buf.toString()));
 
         // cast to EchoService
-        EchoService echo = proxy.getProxy(protocol.refer(EchoService.class, URL.valueOf("dubbo://127.0.0.1:9011/" + DemoService.class.getName() + "?client=mina").addParameter("timeout", 3000l)));
+        EchoService echo = (EchoService) service;
```

Comment:
Is this the fix to #3351 . Can you explain why?

Reply:
> Is this the fix to #3351 . Can you explain why?

I've noticed this question, maybe it shoud be fixed in another pr, I've reverted this file.